app/Providers/EventServiceProvider.php

```diff
@@ -55,10 +55,9 @@ public function register()
         });
 
         $this->app->bind(WebsiteUpdater::class, function (Application $app) {
-            return new WebsiteUpdaterThroughGithubActions(
+            return new WebsiteUpdaterThroughWebhook(
                 new HttpClient(),
-                config('website.github.repository'),
-                config('website.github.token'),
+                config('website.webhook.url'),
```

Comment:
Je specificeert hier `url` maar in de `config/website.php` gebruik je `uri`. Eén van de twee graag :)

Reply:
thx fixed, alles wordt URI